romana-install/roles/stack/kubernetes/install-master/tasks/services.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Install service configurations
```

Comment:
It's etcd, kubernetes, and romana network-policy listener in this same task (just copying some files).